task/backend/analytical_storage.go

```diff
@@ -152,18 +153,45 @@ func (as *AnalyticalStorage) FindRuns(ctx context.Context, filter influxdb.RunFi
 		filterPart = fmt.Sprintf(`|> filter(fn: (r) => r.runID > %q)`, filter.After.String())
 	}
 
+	parsedAfterTime := time.Time{}
+	parsedBeforeTime := time.Now()
+	constructedTimeFilter := ""
+	if feature.TimeFilterFlags().Enabled(ctx) {
+		if filter.AfterTime != "" {
+			tmpParsedAfter, err := time.Parse(time.RFC3339, filter.AfterTime)
```

Comment:
a common trick to avoid a temporary variable where the error is returned is something like the following
```suggestion
			var err error // define the error beforehand, if needed
			// make the assignments
			parsedAfterTime, err = time.Parse(time.RFC3339, filter.AfterTime)
```